docs/static/logging.asciidoc

```diff
@@ -48,6 +48,68 @@ The logger is usually identified by a Java class name, such as
 path as in `org.logstash.dissect`.  For Ruby classes, like `LogStash::Outputs::Elasticsearch`,
 the logger name is obtained by lowercasing the full class name and replacing double colons with a single dot.
 
+It's strongly recommended to use the default log4j configuration that's shipped with {ls}, however the
+following section describe in detail how the rolling strategy works, so you have more control on these settings.
+
+In `log4j2.properties` file are defined three appenders to write on log files: one for plain text, one with json format,
+and another to split log lines on per pipeline basis when using the `pipeline.separate_logs` setting.
+These appenders define triggering policies and rollover strategy. The triggering policy determines if a rollover
+should be performed, while the strategy defines how the rollover should be done.
+In default definition are used two triggering policies: time and size. The time policy defines to create one file per day
+while the size policy force to create a new file every time the size surpasses a limit (100 MB by default).
+
+The default strategy defines to roll logs up to a maximum number of 30 files and also specify to execute the delete action on
+files older than seven days.
+This means that for each day, a maximum of 30 files is created. When the 31-st has to be created a rollover happen, the first
+is removed, all the other are renamed, so that 2nd becomes 1st, and 30th becomes 29th, so creating the space for the new file.
+Each file has a date and if the files are older than seven days they are removed on the rollover execution.
```

Comment:
```suggestion
The rollover limit imposes that a maximum of 30 files is created. When the 31-st has to be created a rollover happen the first
is removed - all the other are renamed - so that 2nd becomes 1st and 30th becomes 29th, creating the space for the new file.
Each file will have a date and if the files are older than 7 days (by default) they are removed during rollover.
```